<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>giftmoney</class>
 <widget class="QWidget" name="giftmoney">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>giftmoney</string>
  </property>
  <widget class="QLabel" name="giftMoney1Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gift Money 1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="giftMoney2Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gift Money 2:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>135</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="giftMoney1LineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="giftMoney2LineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>135</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="itemDescriptionLabel">
      <property name="text">
       <string>Enter Item Description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="itemDescriptionLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="itemPriceLabel">
      <property name="text">
       <string>Enter Item Price:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="itemPriceLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>104</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="populateButton">
      <property name="text">
       <string>Populate!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openFileButton">
      <property name="text">
       <string>Open Amounts File!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
